core/src/main/java/org/apache/iceberg/jdbc/JdbcUtil.java

```diff
@@ -27,7 +27,7 @@
 import org.apache.iceberg.relocated.com.google.common.collect.Iterables;
 
 final class JdbcUtil {
-  protected static final String CATALOG_TABLE_NAME = "ICEBERG_TABLES";
+  protected static final String CATALOG_TABLE_NAME = "iceberg_tables";
```

Comment:
What's the specific reason that we need to change it to lower case  ?  Just want to align with the other table names ?  (I read the context from https://github.com/apache/iceberg/issues/2744 but still did not get the point).

Reply:
> What's the specific reason that we need to change it to lower case ? Just want to align with the other table names ? (I read the context from #2744 but still did not get the point).

PostgreSQL's database kernel is case sensitive, eg:  https://www.cnblogs.com/gaofan/p/11398753.html

Reply:
OK,  got it!